quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
```

Comment:
Any particular reasons for the expanded descriptions? I matched this to the existing GitHub steps.